### PR TITLE
CATROID-791 Remove firebase components

### PR DESCRIPTION
**Pull request by adnaribo**
Deleting all firebase, crashlytics and fabric components. https://jira.catrob.at/browse/CATROID-791

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/build.gradle [hunk 1]
```diff
@@ -108,21 +104,10 @@ ext.copyGoogleServicesFile = { flavorName, flavorId ->
         into "src/$flavorName"
         filter { line ->
             line.replaceAll('@appId@', flavorId)
-                .replaceAll('@apiKey@', getProperty('firebaseApiKey', '@apiKey@'))
```
Comment (danielzhang130) [inline catroid/build.gradle:111]: I think firebaseApiKey is still used by google analytics, unless you want to remove that as well.
So, google-services.json is also still needed.

https://developers.google.com/analytics/devguides/collection/android/v4/#set_up_your_project

**Review by danielzhang130 — CHANGES_REQUESTED** [review]
`:catroid:compileCatroidDebugAndroidTestJavaWithJavac` gradle task didn't compile because the tests still reference the old crashlytics stuff.

- SettingsFragmentCrashReporterTest
- CrashReporterTest
- BaseExceptionHandlerTest

`SettingsFragment#SETTINGS_CRASH_REPORTS` is no longer needed once the tests are corrected.

File: catroid/build.gradle (hunk 1 above)
Comment (wslany) [reply catroid/build.gradle:111]: Google analytics shall be deleted as well.

Comment (adnaribo) [reply catroid/build.gradle:111]: then the whole google-services.json file may be deleted, right? if we dont use google analytics?

**Pull request closed without merging**